protocol/x/blocktime/types/keys.go

```diff
@@ -19,4 +19,7 @@ const (
 
 	// PreviousBlockInfoKey defines the key for PreviousBlockInfo
 	PreviousBlockInfoKey = "PreviousBlockInfo"
+
+	// SynchronyParamsKey defines the key for the SynchronyParams
+	SynchronyParamsKey = "SP:"
```

Comment:
nit: Any reason not to just use the full string `SynchronyParams`?

Reply:
Conventionally we prefer more succinct keys to more efficiently compute them, so it's the other keys are breaking conventions a bit.